test/fixtures/activity.cancel.sm.repro.json

```diff
@@ -1,199 +1,323 @@
 {
-  "events": [
-    {
-      "eventId": 1,
-      "eventTime": "2020-07-30T00:30:02.971655189Z",
-      "eventType": "WorkflowExecutionStarted",
-      "version": -24,
-      "workflowExecutionStartedEventAttributes": {
-        "workflowType": {
-          "name": "ActivityCancelRepro"
-        },
-        "taskQueue": {
-          "name": "tq-1"
-        },
-        "workflowRunTimeout": "10s",
-        "workflowTaskTimeout": "1s",
-        "identity": "97228@samar-C02XG22GJGH6@"
-      }
+ "events": [
```

Comment:
Regenerated this history because it failed the replayer for two reasons 

1. The activities `name` are wrong, this just seems like an issue with the history 

2. The history was missing multiple events, I think because the history came from a WF that timed out. Unclear if that was intentional. Even if it was intentional it is not obvious this history should pass the replayer since the replayed histroy will not match the history file.